addons/web_linkedin/controllers/main.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+import simplejson
+import urllib2
+import werkzeug
+
+from openerp.addons.web import http
+from openerp.addons.web.http import request
+
+
+class Linkedin(http.Controller):
+
+    @http.route("/linkedin/authentication", type='http', auth="user")
```

Comment:
If this is a callback for linkedin server, are you convinced it will work with `auth="user"`?